server/models/providers.go

```diff
@@ -256,6 +256,8 @@ const (
 	UsersProfile Feature = "users-profile"
 
 	UsersIdentity Feature = "users-identity"
+
+	UsersKeys Feature = "" //To be filled...
```

Comment:
```suggestion
	UsersKeys Feature = "users-keys"
```

Reply:
I have added the necessary support to expose this capability in remote provider. 👍